src/ralph/models/selector.py

```diff
@@ -70,7 +68,11 @@ def __init__(self, module="ralph.models.edx"):
         """Instantiates ModelSelector."""
 
         for model in import_module(module).__dict__.values():
-            if isinstance(model, type) and model.__selector__:
+            if (
+                isinstance(model, type)
+                and issubclass(model, BaseModel)
+                and model.__selector__
+            ):
```

Comment:
In fact it should be `object` instead of `model` as it could be anything from the module.

Reply:
You are right, will update that, thanks)